charts/invenio/values.yaml

```
@@ -99,12 +99,11 @@ invenio:
   ## @param invenio.extraConfig Extra environment variables (templated) to be added to all the pods.
   ##
   extraConfig: {}
+  ## @param invenio.extra_env_from_secret DEPRECATED: invenio.extraEnvFromSecret instead
```

```suggestion
  ## @param invenio.extra_env_from_secret DEPRECATED: Use `invenio.extraEnvFrom` or `invenio.extraEnvVars` instead.
```

This this is a typo?